app/boards/shields/nice_view/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+if(CONFIG_ZMK_DISPLAY AND CONFIG_NICE_VIEW_WIDGET_STATUS)
+  zephyr_library_include_directories(${CMAKE_SOURCE_DIR}/include)
+  zephyr_library_sources(custom_status_screen.c)
+  target_sources(app PRIVATE widgets/bolt.c)
```

Comment:
Why not use `zephyr_library_sources` consistently here?

Reply:
Not sure why, fixed.